scripts/skills/special/rf_poise_effect.nut

```diff
@@ -0,0 +1,142 @@
+/*
+Internally we calculate the current threshold inversely. That makes it much cleaner in code to handle increases and reduction of PoiseMax.
+Our intended behavior is that a change in PoiseMax also changes the Poise of that character by the same amount:
+
+The PoiseMax is purely handled in the Properties while the current Poise is handled in this skill.
+It is much cleaner when a change to PoiseMax can solely focus on changing variables in Properties.
+*/
+
+this.rf_poise_effect <- ::inherit("scripts/skills/skill", {
+	m = {
+		// Config
+		HeadshotMult = 1.5,	// multiplier for incoming poise damage on a hit to the head (unless steelbrow was learned)
```

Comment:
Perhaps it would be better if this is a character property similar to `BodyPartDamageMult` as an array so that the body/headshot mult can be defined on a per-character basis in their properties and by default the body can be 1.0 and head can be 1.5.